File: src/test/java/integration/mtp/BaseMtpTest.java
package integration.mtp;

import com.codeborne.selenide.Configuration;
import integration.BaseIntegrationTest;
import org.junit.jupiter.api.BeforeAll;

public abstract class BaseMtpTest extends BaseIntegrationTest {
    @BeforeAll
    static void setUpBaseUrl() {
        Configuration.baseUrl = "https://mtp.es";
    }
}

File: src/test/kotlin/com/github/qky666/selenidepom/test/kotlin/mtp/BaseMtpTest.kt
package com.github.qky666.selenidepom.test.kotlin.mtp

import com.codeborne.selenide.Configuration
import com.github.qky666.selenidepom.test.kotlin.BaseIntegrationTest
import org.junit.jupiter.api.BeforeAll

abstract class BaseMtpTest : BaseIntegrationTest() {
    companion object {
        @BeforeAll
        @JvmStatic
        fun setUpBaseUrl() {
            Configuration.baseUrl = "https://mtp.es"
        }
    }
}
